lib/enumerable/helpers.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Enumerable
+  def check_valid(input, parameter)
+    input.instance_of?(parameter) if parameter.is_a? Class
+    if parameter.is_a? Regexp
+      return false if input.is_a?(Numeric)
+
+      return parameter.match(input)
+    end
+    (input == parameter)
+  end
+
+  def verify_input(initial)
+    start = initial.nil? ? 1 : 0
+    start
+  end
+
+  def multiply_els(arr, multiplier)
+    arr.my_inject(multiplier) { |memo, val| memo * val }
+  end
```

Comment:
```
# Same using a block and inject
(5..10).inject { |sum, n| sum + n }            #=> 45
# Multiply some numbers
(5..10).inject(1) { |product, n| product * n } #=> 151200

.....
(5..10).inject(5) { |product, n| product * n }
(5..10).inject(0) { |product, n| product * n }


```

Inject should pass those cases

Reply:
Those are already taken care

![image](https://user-images.githubusercontent.com/36519478/68626317-2251ed00-04a9-11ea-8d12-f388822f6873.png)